rero_ils/modules/items/api.py

```diff
@@ -397,7 +396,7 @@ def last_location_pid(self):
 
         of the last loan.
         """
-        loan_location_pid = get_last_transaction_location_for_item(self.pid)
+        loan_location_pid = get_last_transaction_loc_for_item(self.pid)
```

Comment:
why changing the name as it is less explicit

Reply:
very long name ( for pep8)